introduction.md

```diff
@@ -1,6 +1,6 @@
 Ruby test sj
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?